inst/@sym/horzcat.m

```diff
@@ -117,7 +116,7 @@
 %! q = sym(ones(1, 0));
 %! assert (isequal ([v q], v))
 
-%!error <ShapeError>
+%!error
```

Comment:
Can we keep some <text> here?  Does it not pass with `<ShapeError>` here?  I thought it would as the error should still include the words `ShapeError`